src/frontend/components/InstructorWrapper/InstructorWrapper.tsx

```diff
@@ -1,17 +1,20 @@
 import * as React from 'react';
 
 import { appState } from '../../types/AppData';
-import { AppDataContext } from '../App/App';
 import { InstructorView } from '../InstructorView/InstructorView';
 
-export const InstructorWrapper = (props: { children: React.ReactNode }) => (
-  <AppDataContext.Consumer>
-    {({ state }) =>
-      state === appState.INSTRUCTOR ? (
-        <InstructorView>{props.children}</InstructorView>
-      ) : (
-        props.children
-      )
+export interface InstructorWrapperProps {
```

Comment:
This interface is internal to this class, I don;t think it's needed to export it. This is true for all interfaces exported but only use internally